cheat.php

```diff
@@ -153,6 +161,15 @@
 	$WaitTimeBeforeFirstScan = 50 + ( 50 - $SkippedLagTime );
 	$PlanetCheckTime = microtime( true );
 
+	if ( $LocalScriptHash === $RepositoryScriptHash )
+	{
+		$RepositoryScriptHash = GetRepositoryScriptHash( );
+	}
+	else
```

Comment:
Hm I was wrong, this can't be `else`, because if version changes, it will only print on next loop.

So change this `else` to `if ( $LocalScriptHash !== $RepositoryScriptHash )`